packages/bson-transpilers/test/yaml/native.yaml

```diff
@@ -963,6 +1048,7 @@ tests:
           csharp: new DateTime(1995, 12, 17, 0, 0, 0).ToString("ddd MMM dd yyyy HH':'mm':'ss UTC")
           object: Date(1995, 11, 17)
           ruby: Time.utc(1995, 12, 17, 0, 0, 0).strftime('%a %b %d %Y %H:%M:%S %Z')
+          rust: Utc.ymd(1995, 12, 17).and_hms(0, 0, 0).to_rfc2822()
```

Comment:
Unfortunately we don't currently have a way to do this with `bson::DateTime`, though we probably should. I filed https://jira.mongodb.org/browse/RUST-1198 to cover the work for that, maybe in the meantime we just omit these examples?

Reply:
Actually, we can pull the same trick as js - emit the date as a string and parse from that.  More direct constructors would still certainly be a good to have.